internal/build/ocpbuild/maker.go

```diff
@@ -119,17 +124,17 @@ func (m *maker) MakeBuildTemplate(
 		Type:       buildv1.BuildSourceDockerfile,
 	}
 
-	sourceConfigHash, err := hashstructure.Hash(sourceConfig, nil)
+	sourceConfigHash, err := hashstructure.Hash(sourceConfig, hashstructure.FormatV2, nil)
```

Comment:
Wondering how this is working? `hashstrucutre` was renamed to `v2`...

Reply:
This is the expected behaviour when the version is the last element in the import path.